Docs/Changelog_sm5_1.txt

```diff
@@ -7,6 +7,17 @@ StepMania 5.1 and StepMania 5.0 have separate changelogs so that devs working
 on the different branches don't have conflicts on the changelog file.
 ________________________________________________________________________________
 
+2016/10/08
+----------
+* [TimingData] Replaced starting point lookup table system with new idea
+  based on transforming timing segments into line segments and using linear
+	interpolation.  Cuts SetOccuranceTimeForAllTaps to 1/3 as much time, which
+	means GetElapsedTimeFromBeat is much faster than before. [kyzentun]
+* [SongPosition] Removed GetWarpBeginRow and GetWarpDestination lua
+  functions.  It's impossible for SongPosition Update to result in a beat
+	that is inside a warp, so they were useless anyway. [kyzentun]
+
```

Comment:
Minor nitpick: the next few entries in this file only have one blank line separating the last field and the previous date.